setup.py

```diff
@@ -75,22 +75,25 @@
     ],
 }
 try:
+    # NOTE(kamo): There packages are not listed if installing from pypi server
```

Comment:
```suggestion
    # NOTE(kamo): These packages are not listed if installing from the PyPI server
```
??? Could you check it?